lab-michael/greeting.js

```diff
@@ -0,0 +1,9 @@
+var myObject  = {};
```

Comment:
Nice work getting this working. There are a few things to note for the next lab that I'd like you to fix:

1. Have a `package.json` file in this directory.
2. Have an entry point; i.e. `index.js`, which is what will be used when we run `node index.js` (this should require in the greet module)
3. Have a `./lib/`  directory that will hold your `greet.js` file (this will be your exported greet function)